circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableReplication.java

```diff
@@ -88,34 +88,38 @@ public void replicate() throws CircusTrainException {
       TableAndStatistics sourceTableAndStatistics = source.getTableAndStatistics(database, table);
       Table sourceTable = sourceTableAndStatistics.getTable();
 
-      PartitionsAndStatistics sourcePartitionsAndStatistics = source.getPartitions(sourceTable,
-          partitionPredicate.getPartitionPredicate(), partitionPredicate.getPartitionPredicateLimit());
+      PartitionsAndStatistics sourcePartitionsAndStatistics = source
```

Comment:
Has anything actually changed in this file other than the formatting?

Reply:
line 106, adding db/table to the arguments

Reply:
OK